changelog/fragments/gitnit.yaml

```
@@ -0,0 +1,13 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      The flag `--git-init` in the new command was deprecated.
```

```suggestion
      The flag `--git-init` in the `new` command was deprecated.
```

Just to avoid confusion